docs/usage/prometheus-metrics.md

```diff
@@ -27,3 +27,16 @@ We provide a {download}`sample Grafana dashboard </_downloads/grafana/liqonetwor
 As presented in the screenshot below, it includes an overview section presenting the overall cross-cluster throughput, followed by detailed per-peering throughput and latency information.
 
 ![Grafana Network Dashboard](/_static/images/usage/prometheus-metrics/network-dashboard.png)
+
+## Virtual kubelet metrics
+
+These metrics are available for each peered remote cluster, providing statistics about the reflected resources:
+
+- **liqo_virtual_kubelet_reflection_item_counter**: the number of reflected resources successfully processed.
```

Comment:
Not clear what "resources" means. I would suggest to rephrase such as:
... the number of reflected resources (e.g., pods, ...)
(list, in the parenthesis, all the objects that are counted)

Reply:
Ok, I will rephrase that. For resources, I meant pods, configmaps, services and others.